src/tests/sim_tests/mod.rs

```diff
@@ -1,12 +1,17 @@
+#[cfg(not(feature = "simulated_input"))]
```

Comment:
It seems better to add this to `src/tests.rs` to exclude the `sim_tests` module.

Reply:
Ok, I thought you could later add sim input tests to `sim_tests`